assets/dev/js/frontend/handlers/image-carousel.js

```diff
@@ -1,4 +1,6 @@
-export default class ImageCarousel extends elementorModules.frontend.handlers.CarouselBase {
+import CarouselBase from './base-carousel';
```

Comment:
What is this file `base-carousel`?

Reply:
@udidol My base-carousel js file for the carousel widgets.